lib/Dialect/Secret/IR/SecretOps.cpp

```diff
@@ -80,10 +82,22 @@ LogicalResult YieldOp::verify() {
 
 void GenericOp::print(OpAsmPrinter &p) {
   ValueRange inputs = getInputs();
-  if (!inputs.empty())
-    p << " ins(" << inputs << " : " << inputs.getTypes() << ")";
 
-  ArrayRef<NamedAttribute> attrs = (*this)->getAttrs();
+  if (!inputs.empty()) {
+    p << " ins(";
```

Comment:
We should also remove `ins` while we're here. 

```mlir
%0 = secret.generic(%arg0: !ty {attr}, %arg1: !ty {attr}) {
    ^body(...
```

I think it should be relatively simple to update the tests and docs for this.

```
sed -i 's/secret.generic ins/secret.generic/g' **/*.mlir **/*.td
```

Reply:
ooh yeah i could totally remove `ins` as well! :)
and no problem. it was somewhat satisfying